### DISCUSSION TITLE
Scheduled Reminders can not use the channel name in Japanese.

### DISCUSSION THREAD

**Discussion by alice7mm** [community/community#33547] [Other Feature Feedback, Questions, & Ideas]
Hi

Our team use a Slack channel named in Japanese.
Scheduled Reminders does not allow us to specify the channel name in Japanese. Because it seems that notifications cannot be sent to channels created in Japanese.

Is there any way to resolve this other than changing the channel name?
If not, please consider supporting multi-byte characters.

**Comment by queenofcorgis** [comment]
Thanks for the feedback! I've labeled this so it can be reviewed by the appropriate team.

**Comment by say8425** [comment]
Same here. Korean also can not be used for a channel name.
